crates/ruff_linter/resources/test/fixtures/airflow/AIR302_context.py

```diff
@@ -0,0 +1,19 @@
+from airflow.decorators import task
+
+@task
+def print_config(**context):
```

Comment:
There are some more things we should check

1. Named keyword arguments (e.g. `def print_config(execution_date)`)
2. Getting the context with `get_current_context` instead of function arguments.
3. `context` in an operator’s `execute`.

These can be added in separate PRs instead after this is merged.

Reply:
I have added logic for other ways to access context value as well. It is part of tests.

Reply:
For `Named keyword arguments (e.g. def print_config(execution_date))` I can create a separate PR